src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java

```diff
@@ -521,7 +522,8 @@ static void closeWebDriver(String userPersona,
                                @NotNull
                                Driver driver) {
         String browserNameForUser = Drivers.getBrowserNameForUser(userPersona);
-        String logFileName = getBrowserLogFileNameFor(userPersona, Platform.web.name(),
+        String platformName = Runner.getPlatform().name();
```

Comment:
why are we changing this? This class is supposed to be only for web execution

Reply:
For electron platform it will give null pointer error for below method
void assignNewPersonaForUser(String userPersona, String newUserPersona) {
        replaceCapabilitiesFor(userPersona, newUserPersona);
        replaceDriverFor(userPersona, newUserPersona);
        replaceAppNameFor(userPersona, newUserPersona);
        replacePlatformFor(userPersona, newUserPersona);
        replaceLogFileNameFor(userPersona, newUserPersona);
    }

For replaceLogFileNameFor method we have below line which will always take web as platform if by default provided
String currentKey = getKeyNameForDeviceLogs(userPersona, currentPlatform.name());